src/test/kotlin/io/github/mojira/arisa/ExecutorTest.kt

```diff
@@ -24,7 +23,7 @@ private val failedModuleRegistryMock = mockk<ModuleRegistry>()
 private val moduleExecutorMock = mockk<ModuleExecutor>()
 private val failedModuleExecutorMock = mockk<ModuleExecutor>()
 
-private val dummyTimeframe = TicketQueryTimeframe(Instant.now(), Instant.now(), true)
+private val dummyTimeframe = ExecutionTimeframe(Instant.now(), Instant.now(), true)
```

Comment:
Would be nice to see if we can add tests for all the new classes now that they are separate, that would avoid most of the errors we are having lately

Reply:
Yeah, adding tests for all these classes would be pretty important. I'm not sure if I'll have enough time this week to add some for this. If not we should definitely keep it in mind.

FWIW, I've tested this all locally in a dry run and it seems to work exactly as before, so it should be fine?

At the very least I didn't have to remove any tests this time 😅